src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/get_options_list_control_factory.tsx

```diff
@@ -76,12 +80,11 @@ export const getOptionsListControlFactory = (): DataControlFactory<
         initialState.sort ?? OPTIONS_LIST_DEFAULT_SORT
       );
 
-      /** Creation options state - cannot currently be changed after creation, but need subjects for comparators */
-      const placeholder$ = new BehaviorSubject<string | undefined>(initialState.placeholder);
-      const hideActionBar$ = new BehaviorSubject<boolean | undefined>(initialState.hideActionBar);
-      const hideExclude$ = new BehaviorSubject<boolean | undefined>(initialState.hideExclude);
-      const hideExists$ = new BehaviorSubject<boolean | undefined>(initialState.hideExists);
-      const hideSort$ = new BehaviorSubject<boolean | undefined>(initialState.hideSort);
+      const placeholder = initialState.placeholder;
```

Comment:
Nice to see these no longer required to be subjects as they cannot change over the lifetime of the control.